### PR TITLE
[nikobus] fixed refresh handling

### PR DESCRIPTION
**Pull request by crnjan**
* fixed refresh handing due `channelLinked` not called anymore on startup for example (similar to https://github.com/openhab/openhab-core/issues/1707),
* "Impacted Modules" can be empty if button is configured as "standalone" and does not impact modules,
* minor doc fix

Signed-off-by: Boris Krivonog boris.krivonog@inova.si

### CODE REVIEW

File: bundles/org.openhab.binding.nikobus/src/main/java/org/openhab/binding/nikobus/internal/handler/NikobusPushButtonHandler.java [hunk 1]
```diff
@@ -118,27 +118,30 @@ public void initialize() {
...
+                }
+
+                String[] impactedModulesString = impactedModulesObject.toString().split(",");
+                for (String impactedModuleString : impactedModulesString) {
+                    ImpactedModuleUID impactedModuleUID = new ImpactedModuleUID(impactedModuleString.trim());
+                    ThingTypeUID thingTypeUID = new ThingTypeUID(bridgeUID.getBindingId(),
+                            impactedModuleUID.getThingTypeId());
+                    ThingUID thingUID = new ThingUID(thingTypeUID, bridgeUID, impactedModuleUID.getThingId());
+                    impactedModules.add(new ImpactedModule(thingUID, impactedModuleUID.getGroup()));
+                }
```
Comment (cpmeister) [inline bundles/org.openhab.binding.nikobus/src/main/java/org/openhab/binding/nikobus/internal/handler/NikobusPushButtonHandler.java:136]: Is it expected for the impactedModules to contain ThingUIDs for things that might not actually exist?
What is the expected behavior if the config contains a thing that doesn't exist?
You can check the validity of a thingUID by iterating through all of the things under a bridge using `getBridge().getThings()`.

That said I don't know what your requirements are here.

Comment (crnjan) [reply bundles/org.openhab.binding.nikobus/src/main/java/org/openhab/binding/nikobus/internal/handler/NikobusPushButtonHandler.java]: Great suggestion, fixed!

**Comment by crnjan** [comment]
Also removes StringUtils (see #7722).

File: bundles/org.openhab.binding.nikobus/src/main/java/org/openhab/binding/nikobus/internal/utils/CRCUtil.java [hunk 2]
```diff
@@ -35,7 +37,8 @@
      *            String representing hex numbers.
      * @return input string + CRC.
      */
-    public static String appendCRC(String input) {
+    @Nullable
+    public static String appendCRC(@Nullable String input) {
```
Comment (cpmeister) [inline bundles/org.openhab.binding.nikobus/src/main/java/org/openhab/binding/nikobus/internal/utils/CRCUtil.java:41]:
```suggestion
    public static @Nullable String appendCRC(@Nullable String input) {
```

**Review by cpmeister — CHANGES_REQUESTED** [review]

**Review by cpmeister — APPROVED** [review]

**Pull request merged**